src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -27,7 +29,8 @@ internal TagHelperExecutionContext(string tagName)
                    uniqueId: string.Empty,
                    executeChildContentAsync: async () => await Task.FromResult(result: true),
                    startWritingScope: () => { },
-                   endWritingScope: () => new StringWriter())
+                   endWritingScope: () => new StringWriter(),
+                   parentItems: new Dictionary<object, object>())
```

Comment:
same issues with names and descriptions exist in this class as in `TagHelperContext`